Gutenberg/cocoapods_helpers.rb

```diff
@@ -172,3 +177,27 @@ def react_native_version!(gutenberg_path:)
 
   package_json_version.split('.').map(&:to_i)
 end
+
+# A workaround for the issue described at
+# https://github.com/wordpress-mobile/WordPress-iOS/pull/21504#issuecomment-1789466523
+#
+# For some yet-to-discover reason, something in the process installing the pods
+# using local sources messes up the LIBRARY_SEARCH_PATHS.
+def workaround_broken_search_paths
+  project = Xcodeproj::Project.open('WordPress/WordPress.xcodeproj')
```

Comment:
There might be a way to read this setting from CocoaPods, but I don't think we need to worry about being that flexible here, right?